### PR TITLE
Unify AppDomain across aot/not-aot

### PR DESCRIPTION
**Pull request by DnlHarvey**
With the changes in the targeting packs, these are no longer necessary.

### CODE REVIEW

**Comment by danmosemsft** [comment]
If you do "msbuild src\shims\apicompat.proj /t:rebuild" it should edit another baseline you can commit too

**Comment by joperezr** [comment]
> Total Issues: 5302

This is definitely not right, I believe that you got this because you didn't pass in a /p:TargetGroup=uapaot. I wouldn't expect any changes to this file at all.

---
Refers to: src/shims/ApiCompatBaseline.netcoreapp.netfx461.txt:5345 in e22b2b7. [](commit_id = e22b2b7e5eac7ab1f087428ed8fdfe2d0034569d, deletion_comment = False)

**Comment by joperezr** [comment]
> Total Issues: 7329

Same here, this should be undone.

---
Refers to: src/shims/ApiCompatBaseline.netcoreapp.netstandard20.txt:7640 in e22b2b7. [](commit_id = e22b2b7e5eac7ab1f087428ed8fdfe2d0034569d, deletion_comment = False)

**Comment by joperezr** [comment]
> Total Issues: 2369

Same here.

---
Refers to: src/shims/ApiCompatBaseline.netcoreapp.netstandard20Only.txt:2372 in e22b2b7. [](commit_id = e22b2b7e5eac7ab1f087428ed8fdfe2d0034569d, deletion_comment = False)

File: src/System.Runtime.Extensions/src/System/AppDomain.cs [hunk 1]
```diff
@@ -25,7 +24,6 @@ public partial class AppDomain : MarshalByRefObject
 
         public static AppDomain CurrentDomain => s_domain;
 
-#if !uapaot
```
Comment (joperezr) [inline src/System.Runtime.Extensions/src/System/AppDomain.cs]: I did a quick search and it looks like these were the only places where we used the uapaot constant, so now that it's not used anymore, do you mind removing it from the .csproj?

**Pull request merged**